android/app/src/main/java/net/minetest/minetest/GameActivity.java

```diff
@@ -201,7 +203,11 @@ public int getDisplayWidth() {
 
 	public void openURI(String uri) {
 		Intent browserIntent = new Intent(Intent.ACTION_VIEW, Uri.parse(uri));
-		startActivity(browserIntent);
+		try {
+			startActivity(browserIntent);
+		} catch (ActivityNotFoundException e) {
+			runOnUiThread(() -> Toast.makeText(this, "No web browser found", Toast.LENGTH_SHORT).show());
```

Comment:
should be in strings.xml

Reply:
Why does that file exist though? I would have guessed for translations, but the strings aren't translated.

Reply:
yes, it's intended for translations.